lms/resources/_js_config/__init__.py

```diff
@@ -203,6 +209,10 @@ def enable_lti_launch_mode(self, course, assignment: Assignment):
         }
         self._config["debug"]["values"] = self._get_lti_launch_debug_values()
 
+        document = self._request.find_service(
+            DocumentService
+        ).get_document_from_assignment(assignment)
+        self._config["document"] = {"filename": document.filename}
```

Comment:
Only exposing the filename where, `lms_url` could be interesting in the future but not needed now.

Same for other attributes like `type`. Let's stick the minimum interface for now.